Min/Max Formula for Metrics Queries Update (#20156)
DataDog/documentation: content/en/metrics/explorer.md

>   - link: "/metrics/distributions/"

Reviewer: @estherk15  distributions shouldn't be linked as a subpage under metrics explorer . It should be moved back as its own page like summary or metrics without limits is. I'd prefer if it existed right under metric types in terms of ordering

Reviewer 2: @ijkaylin I can do that! To clarify you want it at the same level under Metric Type? Or under as a child of Metric Type?

Reviewer 2: https://datadoghq.atlassian.net/jira/software/projects/DOCS/boards/199

Author: Thanks for bringing that up @ijkaylin! 
I made a [request](https://datadoghq.atlassian.net/browse/DOCS-6447) for these changes on the board @estherk15 thank you for your help!

**Note**: The options may differ based on the metric type selected.

### Functions/Formulas

> You can optionally add functions to your query using the function button. Not all functions are available for all metric types. See the [querying][6] documentation to learn more about functions and formulas in queries. 

Reviewer 2: ```suggestion
You can optionally add functions to your query using the function button. Not all functions are available for all metric types. For more information, see the [querying][6] documentation. 
```
